extramodules/it/eng/jhove/module/png/PngModule.java

```diff
@@ -254,26 +271,26 @@ public PngModule() {
 
 		keywordList = new HashMap();
 
-		keywordList.put("Title",
-						new Booolean(false,"Title"));           //  Short (one line) title or caption for image
-		keywordList.put("Author",
-						new Booolean(false,"Author"));          //  Name of image's creator
-		keywordList.put("Description",
-						new Booolean(false,"Description"));     //  Description of image (possibly long)
-		keywordList.put("Copyright",
-						new Booolean(false,"Copyright"));       //  Copyright notice
+		keywordList.put(KEYWORD[0],
+						new Booolean(false,KEYWORD[0]));           //  Short (one line) title or caption for image
+		keywordList.put(KEYWORD[1],
+						new Booolean(false,KEYWORD[1]));          //  Name of image's creator
+		keywordList.put(KEYWORD[2],
+						new Booolean(false,KEYWORD[2]));     //  Description of image (possibly long)
+		keywordList.put(KEYWORD[3],
+						new Booolean(false,KEYWORD[3]));       //  Copyright notice
 		keywordList.put(CREATION_TIME_KEYWORD,
 						new Booolean(false,CREATION_TIME_KEYWORD)); //  Time of original image creation
-		keywordList.put("Software",
-						new Booolean(false,"Software"));        //  Software used to create the image
-		keywordList.put("Disclaimer",
-						new Booolean(false,"Disclaimer"));      //  Legal disclaimer
-		keywordList.put("Warning",
-						new Booolean(false,"Warning"));         //  Warning of nature of content
-		keywordList.put("Source",
-						new Booolean(false,"Source"));          //  Device used to create the image
-		keywordList.put("Comment",
-						new Booolean(false,"Comment"));         //  Miscellaneous comment
+		keywordList.put(KEYWORD[4],
+						new Booolean(false,KEYWORD[4]));        //  Software used to create the image
+		keywordList.put(KEYWORD[5],
+						new Booolean(false,KEYWORD[5]));      //  Legal disclaimer
+		keywordList.put(KEYWORD[6],
+						new Booolean(false,KEYWORD[6]));         //  Warning of nature of content
+		keywordList.put(KEYWORD[7],
+						new Booolean(false,KEYWORD[7]));          //  Device used to create the image
+		keywordList.put(KEYWORD[8],
+						new Booolean(false,KEYWORD[8]));         //  Miscellaneous comment
```

Comment:
To maintain code readability it may be better to replace the `KEYWORD` array with a named variable for each keyword, in the same manner as the `CREATION_TIME_KEYWORD` above. This would make the code more searchable — allowing us to use the variable name to find the precise location it's referenced — as well as remove the need to cross reference the array index numbers with their values, making it more obvious which keyword is being added to the `keywordList` on each line..

Reply:
Thanks for that, David. That makes a lot of sense as I do more of these. I added another commit replacing the array with named variables.

Reply:
That's beautiful, thank you :)